utilities/trace_analyzer.cc

```diff
@@ -0,0 +1,25 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+#ifndef ROCKSDB_LITE
+#ifndef GFLAGS
+#include <cstdio>
+int main() {
```

Comment:
I think trace_analyzer should go under tools rather than utilities, as this would be similar to, say, ldb/db_bench/sst_dump and other tools.